docs/source/step-by-step.rst

```diff
@@ -1,15 +1,760 @@
 The Step-by-Step Guide to Building an ML Accelerator
 ====================================================
 
-After you have read the :doc:`overview <overview>` of the CFU Playground components and
-:doc:`set up <setup-guide>` your CFU-Playground, it's time to accelerate a model.
+After you have read the :doc:`overview <overview>` of the CFU Playground
+components and :doc:`set up <setup-guide>` your CFU-Playground, it's time
+to accelerate a model. This tutorial will walk through the steps for building
+a basic CFU in your choice of nMigen or Verilog.
 
+-------------------------
+Step 1: Make Your Project
+-------------------------
 
-TODO
-----
+You'll be building an accelerator in your own project folder. Navigate to the
+root of your CFU-Playground directory and enter the following in your terminal:
 
-* Step 2: pick a model. Suggest pdti8
-* Step 3: profile and identify an operation to focus upon
-* Step 4: software specialization
-* Step 5: simple calculation gateware
-* Step 6: moving loops into gateware
\ No newline at end of file
+.. code-block:: bash
+
+    $ cp -r proj/proj_template proj/my_first_cfu
```

Comment:
Consider:

- making clear that the reader should pick their own project name
- adding proj/my_first_cfu to the .gitignore file with a comment